STYLE_GUIDE.md

```diff
@@ -36,12 +36,32 @@ pre-commit run isort --all-files  # for isort
 The configuration is stored in [pyproject.toml](pyproject.toml).
 
 
+### 2.4. Code Structure
+
+As a general rule of thumb,
+
+- Don't: use classes to make code look more like an outline
```

Comment:
what does this mean?

Reply:
Lol, good point. I think it had something to do with the last bullet, but yea let me remove it. Copied it from https://www.notion.so/Codebase-74ccd984a16c4095a368806225a555b4#23cafb24f4344751a700cd875dc095e9

Reply:
Fixed